build fail looks related

```

======================================================================
ERROR: test_large_rewind (corehq.apps.hqadmin.tests.test_utils.TestPillowCheckpointSeqStore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/hqadmin/tests/test_utils.py", line 61, in test_large_rewind
    pillow_seq_store()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/hqadmin/utils.py", line 21, in pillow_seq_store
    'pillow checkpoint seq': checkpoint['seq'],
TypeError: 'PillowCheckpoint' object has no attribute '__getitem__'
----------------------------------------------------------------------
Ran 251 tests in 1148.249s
```

--- a/corehq/apps/hqadmin/tests/test_utils.py
+++ b/corehq/apps/hqadmin/tests/test_utils.py
@@ -15,6 +15,7 @@ class DummyPillow(BasicPillow):
 
 @override_settings(PILLOWTOPS={'test': ['corehq.apps.hqadmin.tests.test_utils.DummyPillow']})
 class TestPillowCheckpointSeqStore(TestCase):
+    dependent_apps = []
 
     def setUp(self):
         self.pillow = DummyPillow()

--- a/corehq/apps/hqadmin/utils.py
+++ b/corehq/apps/hqadmin/utils.py
@@ -18,7 +18,7 @@ def pillow_seq_store():
                 message='Found seq number lower than previous for {}. '
                         'This could mean we are in a rewind state'.format(store.checkpoint_id),
                 details={
-                    'pillow checkpoint seq': checkpoint['seq'],
+                    'pillow checkpoint seq': db_seq,
                     'stored seq': store.seq
                 })
         else:

